src/Chapter2.hs

```diff
@@ -336,7 +336,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList first second list = if first < 0 || second < 0 
+  then []
+  else drop first (take (second+1) list)
```

Comment:
Nice solution!

In Haskell, operators are usually surrounded by spaces on both sides: `a + b` instead of `a+b`. This would be especially important, once some additional features of Haskell will be used in module, in order to make the record calls as in other languages: `a.b`.